pkg/configurator/methods.go

```diff
@@ -123,6 +123,18 @@ func (c *client) GetEnvoyLogLevel() string {
 	return constants.DefaultEnvoyLogLevel
 }
 
+// GetLocalProxyMode returns the local proxy mode
+func (c *client) GetLocalProxyMode() configv1alpha2.LocalProxyMode {
```

Comment:
Let's drop these methods in the pkg and use `GetMeshConfig().LocalProxyMode` directly where applicable.

Reply:
I can remove it; I don't have any strong opinions, but is there a definitive reason why?

Reply:
So initially these methods were added as helpers to parse a ConfigMap instead of a MeshConfig resource, so it made sense to have these helpers convert the data type from the string type. With MeshConfig, we can directly access the fields as `GetMeshConfig().<field>`. The refactoring of `pkg/configurator` to migrate from ConfigMap backing to MeshConfig didn't clean up the existing code, so we see this inconsistency. For new fields, doesn't make sense to create a method per field in the interface.

Reply:
Ah makes perfect sense. Thanks for the context!